daq/network.py

```diff
@@ -343,12 +343,13 @@ def _generate_behavioral_config(self):
             LOGGER.info('Waiting %ds for network to settle', self._settle_sec)
             time.sleep(self._settle_sec)
 
-    def direct_device_traffic(self, device, port_set):
+    def direct_device_traffic(self, device):
```

Comment:
I think this is gonna break FOT. By setting the arg port_set to None, the function cleans up the remote_tap.

Reply:
Ok, fixed.  Verified with the new fot tests that check for this case as well.